crytic_compile/crytic_compile.py

```diff
@@ -68,8 +68,9 @@ def _extract_libraries(libraries_str: Optional[str]) -> Optional[Dict[str, int]]
     matches = re.findall(pattern, libraries_str)
```

Comment:
@montyly Is limiting the range to 0x00-0xff purposeful or can this be modified?

Reply:
what do you want to use instead ?

Reply:
I think `{2,20}` would allow what the OP was attempting to use